csharp/private/net/BUILD

```diff
@@ -2,8 +2,6 @@ load("@d2l_rules_csharp//csharp/private:rules/imports.bzl", "import_multiframewo
 
 import_multiframework_library(
     name = "Accessibility",
-    visibility = ["//visibility:public"],
-
```

Comment:
This is what I meant by the buildifier-ish changes. I didn't feel like fighting the tools so these changes are in the first commit.